### DISCUSSION TITLE
Titiler Performance Test

### DISCUSSION THREAD

**Discussion by zx128** [developmentseed/titiler#627] [Show and tell]
Results of performance tests for titiler are posted here. They are done by GIBS/VisLabs team.

**Comment by vincentsarago** [comment]
@zx128 ?

**Comment by zx128** [comment]
Test done on a bare metal host:

![part1](https://user-images.githubusercontent.com/54592015/231009801-a7f12c4a-6f7f-4f4d-bdd9-beec82eeb0f9.jpg)

![part2](https://user-images.githubusercontent.com/54592015/231009925-17ff0189-2067-480a-98b5-890a2f364e23.jpg)

![part3](https://user-images.githubusercontent.com/54592015/231009982-e2374448-ad53-4dd7-bd30-3a9825782506.jpg)

![part4](https://user-images.githubusercontent.com/54592015/231010027-11629e13-d5fb-47d0-9ef2-2aea39d2085f.jpg)

**Reply by vincentsarago** [reply]
It seems you are testing titiler-pgstac not titiler itself. Can you move the discussion to the appropriate repo 🙏

It would also be nice to share the method so we can reproduce it.

Thank you

**Reply by vincentsarago** [reply]
also note that we already have some benchmark in titiler-pgstac (which could be enhanced https://github.com/stac-utils/titiler-pgstac/tree/master/benchmark

**Reply by vincentsarago** [reply]
it would also be interesting to share the environment variables set during your tests because it can have a huge impact 
see https://github.com/stac-utils/titiler-pgstac/blob/b510490d0fc5764474493e206884773592a42f53/docker-compose.yml#L69-L107

**Reply by zx128** [reply]
Sure, I will check these links and provide more info about our tests.

**Comment by zx128** [comment]
Here are some results for tests in AWS:


![aws](https://user-images.githubusercontent.com/54592015/231218391-b93eb834-b09a-4d67-a8da-531fd141a55c.png)
